content/kubeone/master/tutorials/creating_clusters/_index.en.md

```diff
@@ -477,7 +477,7 @@ cloudProvider:
 **Make sure to replace the placeholder values with real values in the
 cloud-config section.**
 
-In the [Kubermatic documentation](https://docs.kubermatic.com/kubermatic/v2.15/requirements/cloud_provider/azure/)
+In the [Kubermatic documentation](https://docs.kubermatic.com/kubeone/master/architecture/requirements/machine_controller/azure/azure/)
```

Comment:
```suggestion
In the [Kubermatic documentation]({{< ref "../../architecture/requirements/machine_controller/azure/azure/" }})
```

please replace direct links with ref macro, that will fix links permanently (since it will break to compile if the linked page will no loner exist)

Reply:
taken care.. replaced with ref macro. ready for review!